runtime/local/src/lib.rs

```diff
@@ -1854,7 +1854,6 @@ impl_runtime_apis! {
         }
     }
 
-    #[cfg(feature = "evm-tracing")]
```

Comment:
What would be the consequence if we did this for all runtimes, except for a slightly bigger wasm blob?

Reply:
pretty much

Reply:
Something to keep in mind I guess for the future - don't see any particular issue in enabling it be default if that's the case.